tools/run_tests/run_python.sh

```diff
@@ -40,4 +40,6 @@ export DYLD_LIBRARY_PATH=$ROOT/libs/$CONFIG
 export PATH=$ROOT/bins/$CONFIG:$ROOT/bins/$CONFIG/protobuf:$PATH
 source "python"$PYVER"_virtual_environment"/bin/activate
 
-"python"$PYVER $GRPCIO_TEST/setup.py test -a "-n8 --cov=grpc --junitxml=./report.xml --timeout=300 -v --boxed --timeout_method=thread"
+mkdir -p reports
+
+"python"$PYVER $GRPCIO_TEST/setup.py test -a "-n8 --cov=grpc --html=$ROOT/reports/report.html --junitxml=$ROOT/report.xml --timeout=300 -v --boxed --timeout_method=thread"
```

Comment:
I'm not sure about the XML report, but the HTML report definitely has to go into a subdirectory of `reports`. All of those reports directories are merged when doing the complete coverage report. It should also have an `index.html` file so that it will be linked to properly.